sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveQuerySuite.scala

```diff
@@ -1187,6 +1187,22 @@ class HiveQuerySuite extends HiveComparisonTest with SQLTestUtils with BeforeAnd
       }
     }
   }
+
+  test("Wrong Hive table statistics may trigger OOM if enables join reorder in CBO") {
```

Comment:
IMHO you can just test the read logic for Hive's stats properties in `StatisticsSuite`, instead of a end-to-end test case, developers may not know what's going on by this test case.